<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PALADIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Garamond&quot;;
background-color: rgb(30, 58, 138);
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTextbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
background-color: rgb(44, 44, 44);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../photos/paladin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
text-decoration: underline;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font-size:33pt;
font-weight: 700;

font: 75 8pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Don't have an account? Create one now!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noAccountButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0); /* Przezroczyste t≈Ço */
    border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
